### PR TITLE
Adding DPIF-R3 forms

### PR DESCRIPTION
**Pull request by albertkol**
### How to test
1. Be on latest docker-runner and all other repos
2. Git checkout `run-dpif-r3` branch on:
- digital-form-builder-adapter
- funding-service-pre-award-stores
- funding-service-design-frontend
- funding-service-design-notification
3. `make up`
4. Annoying extra step because open date is in the future:
```
docker exec funding-service-pre-award-stores-1 python -m fund_store.scripts.amend_round_dates -q update-round-dates-fab --fund_short_name DPIF --round_short_name R3 -o PAST -d FUTURE
```
5. Go to https://frontend.levellingup.gov.localhost:3008/funding-round/DPIF/R3

### CODE REVIEW

**Review by nuwan-samarasinghe — DISMISSED** [review]
Hope these all forms are tested 

LGTM

File: fsd_config/form_jsons/dpif_r3/dpif-declarations.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+            "section": "HgUinB",
+            "controller": "./pages/summary.js"
+        },
+        {
+            "path": "/declarations",
+            "title": "Declarations",
+            "components": [
+                {
+                    "type": "Html",
+                    "content": "<p class=\"govuk-body govuk-!-font-weight-bold\">In this section, we'll ask you to:</p>\n<ul class=\"govuk-list govuk-list--bullet\">\n<li>confirm you\u00a0have signed the Local Digital Declaration and agree to follow the 5 principles</li>\n<li>confirm your section 151 officer consents to the funds being carried over and spent in the next financial year (March 2025 to April 2026) and beyond, if deemed necessary in project budget planning</li>\n<li>confirm you agree to let all outputs from this work be published under open licence with a view to any organisation accessing, using or adopting them freely</li>\n<li>confirm the information you have provided is accurate</li>\n</ul>",
```
Comment (gidsg) [inline fsd_config/form_jsons/dpif_r3/dpif-declarations.json:18]: Is this unicode character intentional? `\u00a0`?

Comment (albertkol) [reply fsd_config/form_jsons/dpif_r3/dpif-declarations.json:18]: No idea, this is FAB's export..

@nuwan-samarasinghe @NarenderRajuB @wjrm500 any idea?

Comment (albertkol) [reply fsd_config/form_jsons/dpif_r3/dpif-declarations.json:18]: I don't want to delay deployment, as the deadline is on Wednesday. If the unicode char is a problem, Forms team can look into it separately.

**Review by nuwan-samarasinghe — APPROVED** [review]

**Review by NarenderRajuB — APPROVED** [review]
LGTM

**Pull request merged**